### PR TITLE
add: sdk branch

### PR DESCRIPTION
**Pull request by Ankit098**

### CODE REVIEW

File: src/tests/sample-local-test.py [hunk 1]
```diff
@@ -3,36 +3,22 @@
...
-
-def log_contextual_info(desc,loglevel,page):
-    page.evaluate("_ => {}",
-                  "browserstack_executor: {\"action\": \"annotate\", \"arguments\": {\"data\":\"" + desc + "\", \"level\": \"" + loglevel + "\"}}");
 @pytest.mark.xfail()
 def checkTitle(title):
     print("Title =>"+title)
-    assert title() == "BrowserStack Lical"
\ No newline at end of file
+    assert title() == "BrowserStack Lical"
```
Comment (karanshah-browserstack) [inline src/tests/sample-local-test.py:24]: BrowserStack Local

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -13,26 +13,10 @@ Playwright with PyTest Test runner Integration with BrowserStack.
 * To run your automated tests using BrowserStack, you must provide a valid username and access key. This can be done by setting the BROWSERSTACK_USERNAME and BROWSERSTACK_ACCESS_KEY environment variables.
```
Comment (karanshah-browserstack) [inline README.md:13]: Modify this line to change inside browserstack yml

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Ankit098) [reply README.md:13]: cool, either one will work though

File: tests/sample-test.py [hunk 3]
```diff
@@ -24,31 +21,14 @@ def test_sample(session_capabilities, base_url) -> None:
...
         #expect(cart_item.locator(".title")).to_have_text(phone)
         print("Cart item => "+cart_item.locator(".title").all_inner_texts()[0])
         assert cart_item.locator(".title").all_inner_texts()[0]==phone[0]
-
-        # Update the test result
-        mark_test_status("passed", "The cart has " + str(cart_item.locator(".title").all_inner_texts()[0]), page)
     except Exception as err:
         # Extract error message from Exception
         error = str(err).split("Call log:")[0].replace("\n", " but ").replace(":", "=>").replace("'", "")
-        mark_test_status("failed", error, page)
```
[the remark this answers is not in the document]
Comment (Ankit098) [reply tests/sample-test.py]: added for now

**Review by karanshah-browserstack — APPROVED** [review]

**Pull request merged**